packages/kosu-deployed-addresses/src/index.ts

```diff
@@ -32,4 +44,32 @@ const getAddressesForNetwork = (networkId: number | string): KosuAddresses => {
     };
 };
 
-export { DeployedAddresses, getReceiptsForNetwork, getAddressesForNetwork };
+/**
+ * Get the deployment addresses for a desired network by id.
+ *
+ * @param web3Wrapper A `Web3Wrapper` instance to get contract info.
+ * @returns The contract instances.
+ */
+export const getMigratedContractsForNetwork = async (web3Wrapper: Web3Wrapper): Promise<MigratedContracts> => {
```

Comment:
We may want to have these be exported functions, I'm not sure how the doc generation will work here. 

If it generates fine and accurately this is good.

Reply:
That was me getting distracted or undoing on the wrong window.  I finished the thought. (again?)

Reply:
#358

Reply:
Tagged in #358